pallets/runtime/src/bridge.rs

```diff
@@ -214,10 +263,9 @@ decl_module! {
 
         /// Handles an approved signer set multisig account change proposal.
         pub fn handle_relayers(origin, account_id: T::AccountId) -> DispatchResult {
+            ensure!(!Self::frozen(), Error::<T>::Frozen);
```

Comment:
This should be callable via the admin key rather than via a MultiSig proposal (like `change_relayers`)

Reply:
This function allows the relayers themselves to vote on changing the multisig account. @adamdossa if changing the multisig account is only going to be done by the admin, we should simply remove this function.

Reply:
I think we should remove, and allow the admin key only to assign a new a new multisig account. I guess this could also be used to assign a non-multisig (i.e. external) account as well? Perhaps we should rename `relayers` to `controller`.